__tests__/app.test.js

```diff
@@ -266,7 +265,79 @@ describe("GET /api/articles/:article_id/comments", () => {
         .get("/api/articles/hello/comments")
         .expect(400)
         .then(({ body }) => {
-          expect(body.msg).toBe("Something wrong with input");
+          expect(body.msg).toBe("Something wrong with input or body");
+        });
+    });
+  });
+
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("POST 200: posts the user's comment", () => {
+      const newComment = { username: "butter_bridge", comment: "test comment" };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          const { postedComment } = body;
+          expect(postedComment).toBe(newComment.comment);
```

Comment:
You would want to check it has the correct article Id on it, otherwise you cant be sure its been linked to the correct article.